examples/pass-server-data/routes/item.js

```diff
@@ -0,0 +1,16 @@
+const api = require('../operations/get-item')
+
+// Returns the express handler function. Necessary so that we can get the nextApp instance.
+function getItemRenderer (nextApp) {
+  return (req, res) => {
+    console.log('Express is handling request')
+    const itemData = api.getItem()
+
+    res.format({
```

Comment:
Since this is an example, may be it's better to do this inside the "server.js" to keep it simple and short.
And I think we don't need to use `res.format` as well.

do something like this:

```js
server.get('/item', (req, res) => {
  const itemData = api.getItem()
  return app.render(req, res, '/item', { itemData })
})
```